docker/Dockerfile

```diff
@@ -30,17 +25,20 @@ WORKDIR /app/arcade
 # Copy the parent directory contents into the container
 COPY ../dist /app/arcade/
 
+# Copy the toolkits.txt file into the container
+COPY ./docker/toolkits.txt /app/arcade/
+
 # List files for debugging purposes
 RUN ls -lah /app/arcade/
 
-
 # Install the wheel with extras (not evals for now)
 RUN python -m pip install ./arcade_ai-${VERSION}-py3-none-any.whl fastapi
```

Comment:
Theres an issue with this for dev builds. For example if the version is 0.1.0.dev0, when installing toolkits from pypi, it will pull the latest arcade-ai package from pypi and overwrite this version which can have unintended side effects when testing in staging. Theres a quick, but non-ideal fix in main at the moment